cclib/method/electrons.py

```diff
@@ -31,6 +31,20 @@ def __repr__(self):
         """Returns a representation of the object."""
         return "Electrons"
 
+    def alpha(self):
+        """Number of alpha electrons"""
+        homos = self.data.homos
+        if len(homos) == 1:
+            return homos[0] + 1
+        return homos[0] + 1
+
+    def beta(self):
+        """Number of beta electrons"""
+        homos = self.data.homos
+        if len(homos) == 1:
```

Comment:
Since there's a maximum of two elements, is this whole thing equivalent to return self.data.homos[-1] + 1?

Reply:
Oh, that's right. Thanks for the suggestion!